boto/s3/bucket.py

```diff
@@ -852,7 +853,7 @@ def copy_key(self, new_key_name, src_bucket_name,
             if isinstance(src_key_name, bytes):
                 src_key_name = src_key_name.decode('utf-8')
         else:
-            src_key_name = boto.utils.get_utf8_value(src_key_name)
+            src_key_name = get_utf8able_str(src_key_name)
```

Comment:
I think this whole if/else statement can be replaced with `src_key_name = get_utf8able_str(src_key_name)`, right? The if block above is just doing what get_utf8able_str would do for the PY3 case anyway.

Reply:
Looks like it! Cut out the if else block.